torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3161,11 +3161,17 @@ std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::var_mean(
 
 at::Tensor XLANativeFunctions::view_copy_symint(const at::Tensor& self,
                                                 at::SymIntArrayRef sym_size) {
-  // TODO: support symbolic sizes
-  auto size = C10_AS_INTARRAYREF_SLOW(sym_size);
   TORCH_LAZY_FN_COUNTER("xla::");
-  return bridge::AtenFromXlaTensor(tensor_methods::view(
-      bridge::GetXlaTensor(self), XlaHelpers::I64List(size)));
+  c10::optional<at::IntArrayRef> int_sizes =
+      c10::asIntArrayRefSlowOpt(sym_size);
+  bool all_dims_static = int_sizes.has_value();
+  if (all_dims_static) {
+    return bridge::AtenFromXlaTensor(tensor_methods::view(
+        bridge::GetXlaTensor(self), XlaHelpers::I64List(int_sizes.value())));
+  }
+
+  return bridge::AtenFromXlaTensor(
```

Comment:
nit, can we put this in a else branch so it is more clear?

Reply:
done